Source/debug.cpp

```diff
@@ -984,6 +985,55 @@ std::string DebugCmdChangeTRN(const string_view parameter)
 	return out;
 }
 
+std::string DebugCmdOpenAllChests(const string_view /*parameter*/)
+{
+	int countOpened = 0;
+	for (int i = 0; i < MAXOBJECTS; i++) {
+		Object &o = Objects[i];
+		if ((o.IsChest() || o.IsSarcophagus()) && !o.IsOpen()) {
+			OperateObject(*MyPlayer, o);
+			countOpened++;
+		}
+	}
+	return fmt::format("{:d} opened", countOpened);
+}
+
+std::string DebugCmdBreakAllBarrels(const string_view /*parameter*/)
+{
+	int countBroken = 0;
+	for (auto &o : Objects) {

```

Comment:
Both of these loops should iterate over active objects, just in case this is called on a level which doesn't reinitialise the entire array.

Reply:
Will try